crates/precompile/src/bls12_381/g1_msm.rs

```diff
@@ -49,9 +49,9 @@ pub(super) fn g1_msm(input: &Bytes, gas_limit: u64) -> PrecompileResult {
         // Filter out points infinity as an optimization, since it is a no-op.
         // Note: Previously, points were being batch converted from Jacobian to Affine. In `blst`, this would essentially,
         // zero out all of the points. Since all points are in affine, this bug is avoided.
-        if encoded_g1_element.iter().all(|i| *i == 0) {
-            continue;
-        }
+        // if encoded_g1_element.iter().all(|i| *i == 0) {
+        //     continue;
+        // }
```

Comment:
All tests are still passing for this commit, I'll revert it since it is still useful as an optimization to remove the points at infinity